includes/wc-order-item-functions.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * WooCommerce Order Item Functions
```

Comment:
Just a note to myself that I'll want to move some of these queries to the order item data store.